client/packages/electron/src/electron.ts

```diff
@@ -184,22 +188,39 @@ const start = (): void => {
   );
 
   ipcMain.handle(IPC_MESSAGES.CONNECTED_SERVER, async () => connectedServer);
-  ipcMain.handle(IPC_MESSAGES.START_BARCODE_SCAN, () =>
```

Comment:
moved below to one common barcode section, and instead of barcodeScaner, using getCurrentScanner()